scripts/monorepo/find-and-publish-all-bumped-packages.js

```diff
@@ -79,10 +79,10 @@ const findAndPublishAllBumpedPackages = () => {
         process.exit(1);
       }
 
-      const hasSpecificCommitMessage =
-        commitMessage.startsWith(BUMP_COMMIT_MESSAGE);
+      const hasSpecificPublishTag =
+        commitMessage.includes(PUBLISH_PACKAGES_TAG);
```

Comment:
I would maybe keep both logic in `OR`, what do you think?

Reply:
If we keep both, but cli (`npm run bump-all-updated-packages`) will only append `@publish-...` tag, this means that some users might put this commit message manually

I think its better to avoid this and keep cli as the only right way to do packages bumping